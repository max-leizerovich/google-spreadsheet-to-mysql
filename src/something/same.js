export function same() {
    return 'aaa';
}
