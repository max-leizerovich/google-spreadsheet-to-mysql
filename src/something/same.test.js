import { same } from './same';

describe('describe same me', () => {
    it('Should return same', () => {
        const resolved = same();
        expect(resolved).toMatchSnapshot();
    });
});
