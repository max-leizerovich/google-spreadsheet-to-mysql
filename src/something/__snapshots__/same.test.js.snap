// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`describe same me Should return same 1`] = `"aaa"`;
